domino-ui/src/main/java/org/dominokit/domino/ui/forms/SelectOptionGroup.java

```diff
@@ -44,7 +44,7 @@ public SelectOptionGroup(Node titleElement) {
         return create((Node) titleElement);
     }
 
-    public static <T> SelectOptionGroup<T> create(IsElement titleElement) {
+    public static <T, E extends HTMLElement> SelectOptionGroup<T> create(IsElement<E> titleElement) {
```

Comment:
can we make this `public static <T> SelectOptionGroup<T> create(IsElement<?> titleElement) {`
having double generics in the create API sounds like too verbose.